### PR TITLE
[Merged by Bors] - feat(SetTheory/Ordinal/Basic): `ω ≤ a.ord ↔ ℵ₀ ≤ a`

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Ordinal/Basic.lean [hunk 1]
```diff
@@ -1234,6 +1234,10 @@ theorem ord_eq_zero {a : Cardinal} : a.ord = 0 ↔ a = 0 :=
 theorem ord_eq_one {a : Cardinal} : a.ord = 1 ↔ a = 1 :=
   ord_injective.eq_iff' ord_one
 
+@[simp]
+theorem omega0_le_ord {a : Cardinal} : ω ≤ a.ord ↔ ℵ₀ ≤ a := by
+  rw [← ord_aleph0, ord_le_ord]
```
Comment (urkud) [inline Mathlib/SetTheory/Ordinal/Basic.lean:1239]: Do we have lemmas with `ω` on the other side of the inequality? Do we have lemmas about equalities? If no, then please add them.

Comment (vihdzp) [reply Mathlib/SetTheory/Ordinal/Basic.lean:1239]: The "lemma" `a.ord = ω ↔ a = ℵ₀` is false. But `a.ord < ω ↔ a < ℵ₀` is true and trivially provable from this one, so I'll add it.

Comment (urkud) [reply Mathlib/SetTheory/Ordinal/Basic.lean:1240]: I've just pushed a proof of this lemma.

Comment (vihdzp) [reply Mathlib/SetTheory/Ordinal/Basic.lean:1240]: Oh you're right, it's `a.card = ℵ₀ ↔ a = ω` which is wrong.

**Comment by urkud** [comment]
Thanks! Since I've pushed >30% of the code, I won't merge it myself.
maintainer merge

**Review by j-loreaux — COMMENTED** [review]
Thanks!

bors merge

**Pull request closed without merging**